backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -1424,6 +1424,9 @@ class TracimLiveEventHeaderSchema(marshmallow.Schema):
     # TODO - G.M - 2020-05-14 - Add Filtering for text/event-stream mimetype with accept header,
     #  see: https://github.com/tracim/tracim/issues/3042
     accept = marshmallow.fields.String(required=True, load_from="Accept", dump_to="Accept")
+    after_event_id = marshmallow.fields.Int(
+        required=False, example=42, validate=strictly_positive_int_validator
```

Comment:
You should but a `default=0` (or `missing`, I don't remember) as your query method declares to only handle `int`, not `None`.